ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -492,10 +494,14 @@ periodics:
               RUN set -xeuo pipefail && \\
                   mkdir -p /srv/repo /tmp/working && \\
                   curl -L "${tar_url}" | tar xf - -C /srv/repo/ && \\
+                  cp /etc/yum.repos.d/fedora-updates-testing-modular.repo /tmp && \\
```

Comment:
TODO: Host repo files similar to `base-4-x.svc` instead of re-using repo files from cosa container